src/literature/components/PublicationInfo.jsx

```diff
@@ -14,15 +46,7 @@ class PublicationInfo extends Component {
             <span> {info.get('journal_volume')}</span>
           )}
           {info.has('year') && <span> ({info.get('year')})</span>}
-          {info.has('page_start') &&
-            info.get('page_end') && (
-              <span>
-                {' '}
-                {info.get('page_start')}-{info.get('page_end')}
-              </span>
-            )}
-          {info.has('page_start') && <span> {info.get('page_start')}</span>}
-          {info.has('artid') && <span> {info.get('artid')}</span>}
+          {this.displayArtidPageInfoAndJournalIssue()}
```

Comment:
```jsx
<span> {journalIssue}<span>
{pageOrArtidDisplay && <span>, {pageOrArtidDisplay}</span>}
```

Reply:
The comma should be displayed only if there's journal_issue followed by the page or artid info. In this case it might show up even when there's no journal_issue

Reply:
do you think this would be better
```
<span> {journalIssue}<span>
{pageOrArtidDisplay && journalIssue &&& <span>, {pageOrArtidDisplay}</span>}
```

Reply:
in that case pageOrArtid won't show up if journal issue is missing, which can happen I assume
```jsx
{journalIssue && <span> {journalIssue}<span>}
{pageOrArtidDisplay && journalIssue && <span>,</span>}
{pageOrArtidDisplay && <span> {pageOrArtidDisplay}</span>}
```